src/command_modules/azure-cli-appservice/setup.py

```diff
@@ -28,7 +28,8 @@
     'azure-cli-core',
     'azure-mgmt-web==0.31.0',
     # v1.17 breaks on wildcard cert https://github.com/shazow/urllib3/issues/981
-    'urllib3[secure]==1.16'
+    'urllib3[secure]==1.16',
+    'pyOpenSSL==0.16.2',
```

Comment:
Can you remove the version completely from here.

This version no. does not exist.
```
# pip install pyOpenSSL==0.16.2
Collecting pyOpenSSL==0.16.2
  Could not find a version that satisfies the requirement pyOpenSSL==0.16.2 (from versions: 0.6, 0.7, 0.8, 0.9, 0.10, 0.12, 0.13, 0.13.1, 0.14, 0.15, 0.15.1, 16.0.0, 16.1.0, 16.2.0)
No matching distribution found for pyOpenSSL==0.16.2
```